### DISCUSSION TITLE
[1.1.424] Animal openpose support added

### DISCUSSION THREAD

**Discussion by huchenlei** [Mikubill/sd-webui-controlnet#2351] [General]
As requested in https://github.com/Mikubill/sd-webui-controlnet/issues/2258, we added support for animal openpose reprocessor. You can now find it under `Openpose` control type.

![Screen Capture 036 - Stable Diffusion - 127 0 0 1](https://github.com/Mikubill/sd-webui-controlnet/assets/20929282/b0ebd611-8fcd-47ce-8087-cff089c5d4a5)

sd-webui-openpose-editor starts to support edit of animal openpose from version v0.2.4.
![demo (2)](https://github.com/Mikubill/sd-webui-controlnet/assets/20929282/8bba8b1e-8e8a-4bc6-95df-b13634f4d166)

Now you can use your creativity and use it along with other ControlNet models. For example, you can use it along with human openpose model to generate half human, half animal creatures. Here is an example:
![CentaurCtrl](https://github.com/Mikubill/sd-webui-controlnet/assets/20929282/e47f400a-6d17-4ced-8306-f75f3b0dbabd)
![Centaur2](https://github.com/Mikubill/sd-webui-controlnet/assets/20929282/c08ed369-e2ee-4caa-8aae-5ec45ef96a65)
![Centaur](https://github.com/Mikubill/sd-webui-controlnet/assets/20929282/a709faaa-8e86-4b81-80a6-87047d6ae202)

You can post your generations with animal openpose model here and inspire more people to try out this feature. Thanks!

Project repo: https://github.com/abehonest/ControlNet_AnimalPose
Pruned model download: https://huggingface.co/huchenlei/animal_openpose (Original model from project repo is 8GB, I just pruned it and re-upload it.)

Note:
We do notice that it is awkward now to align inputs of multiple control units. https://github.com/Mikubill/sd-webui-controlnet/pull/2346 is on the way to let people align Control images in multiple units more easily.

**Comment by abehonest** [comment]
Thanks for adding my model to webui! I'm glad to see it's more accessible now!

**Comment by x-CK-x** [comment]
@huchenlei is there an easy way of using something like a `union` operation on certain pose parts of this animal model like the `ears & tail` with say an existing pose from the human model (assuming everything lines up), such that it effectively creates a pose for like anime characters that have those animal features per se?

Or is there something more specific I would have to do?

**Reply by huchenlei** [reply]
Unfortunately, animal openpose skeleton does not contain information about ears / tail.

**Reply by huchenlei** [reply]
I would recommend you look into https://github.com/Mikubill/sd-webui-controlnet/discussions/2472 for blending in animal feature on human character with IP-Adapter.

**Comment by magnusroe** [comment]
I have run some videos of my cat through this, it would be very fun if if it could track ears or mouth though. Is it very involved to introduce additional tracked elements?
